src/block_storage/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Sandro-Alessio Gierens <sandro@gierens.de>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Block Storage API implementation bits.
+
+mod api;
+mod protocol;
+mod volumes;
+
+pub use self::protocol::{VolumeSortKey, VolumeStatus};
```

Comment:
You may want to expose VolumeAttachment since it's used in the public API of Volume.

Reply:
done